### ISSUE TITLE
Error in cp-nav logs regarding userid type

### ISSUE THREAD

**Issue by richhayler** [engram-design/CPNav#101]
**Description**
```
2022-05-14 00:29:14 [ERROR] craft\services\UserGroups::getGroupsByUserId(): Argument #1 ($userId) must be of type int, null given, called in /craft/vendor/verbb/cp-nav/src/services/Layouts.php on line 95 - /craft/vendor/craftcms/cms/src/services/UserGroups.php: 165.
2022-05-14 00:29:14 [INFO] Request context:
```

**Additional info**
- Plugin version: 4.0.0
- Craft version: 4.0.2
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/cp-nav` requirement in `composer.json` to:

```
"require": {
  "verbb/cp-nav": "dev-craft-4 as 4.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [4.0.1](https://github.com/verbb/cp-nav/releases/tag/4.0.1)
